packages/tradershub/src/components/GetADerivAccountBanner/GetADerivAccountDialog.tsx

```diff
@@ -21,7 +21,7 @@ const GetADerivAccountDialog = ({ isOpen, onClose }: TGetADerivAccountDialog) =>
 
     return (
         <ReactModal ariaHideApp={false} isOpen={isOpen} shouldCloseOnOverlayClick={false} style={CUSTOM_STYLES}>
-            <div className='w-[calc(100vw-24px)] md:w-[440px] bg-system-light-primary-background p-1200 rounded-400 flex justify-between flex-col gap-1200'>
+            <div className='w-[440px] bg-system-light-primary-background rounded-default flex justify-between flex-col gap-24'>
```

Comment:
@thisyahlen-deriv The padding is missed here.

Reply:
The padding was redundant. Gap was enough